Source/WebInspectorUI/UserInterface/Views/DetailsSection.css

```diff
@@ -252,12 +252,12 @@ body[dir=rtl] .details-section > .header::before {
     margin-inline-start: 5px;
 }
 
-.details-section > .content > .group > .row:has(.warning) {
+.details-section > .content > .group > .row:is(.simple, .font-variation):has(.warning) {
```

Comment:
this kinda seems like a layering violation, no?  why should `WI.DetailsSection` know about `WI.DetailsSectionSimpleRow` or `WI.FontVariationDetailsSectionRow`?